lib/swagger/info.rb

```diff
@@ -0,0 +1,35 @@
+class Swagger::Info
+  attr_accessor :fields
+
+  def initialize(fields)
+    @fields = Hashie::Mash.new(fields)
+  end
+
+  delegate :description,
```

Comment:
Sure, I can do that and get rid of the use of Hashie (which I didn't add as a dependency)